src/pages/Mentorship.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+     <Layout>
+         <b-row>
+      <b-col lg="7" md="7">
+            <h1 class="page-title">{{ $page.main.title }}</h1>
+            <div class="markdown" v-html="$page.main.content" /> 
+         </b-col>
+      <b-col lg="5" md="5">
+
+          <h2>Mentorship Training</h2>
+          <p>We encourage all participants to watch this video on open source mentorship.</p>
+          <iframe width="100%" height="314" src="https://www.youtube.com/embed/6W5najMfcy0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          <p>Video credit: MongoDB</p>
+          <p>More information to guide you can be found in this <a href="https://www.mentorship.guide/" target="_blank">resource</a></p>
+
+      </b-col>
```

Comment:
I am not convinced by having the 2 columns there. I would put it directly in the Markdown, so we can have on the right column the Table of content

"Mentorship Training" may sounds like it is only for mentors

Reply:
Maybe 'Participants' Training' would be better?

Reply:
I will have a look how to add the table of content for all pages

Reply:
> I will have a look how to add the table of content for all pages

Thank you!